src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/HttpsResource.cs

```diff
@@ -19,20 +21,10 @@ protected override int GetPort(ResourceRequestContext context)
 
         protected override void ModifyHost(ServiceHost serviceHost, ResourceRequestContext context)
         {
-            EnsureHttpsCertificateInstalled(context.BridgeConfiguration);
-            base.ModifyHost(serviceHost, context);
-        }
-
-        internal static string EnsureHttpsCertificateInstalled(BridgeConfiguration configuration)
-        {
             // Ensure the https certificate is installed before this endpoint resource is used
-            string certThumbPrint = CertificateManager.InstallMyCertificate(configuration,
-                                                                            configuration.BridgeHttpsCertificate);
-
-            // Ensure http.sys has been told to use this certificate on the https port
-            CertificateManager.InstallSSLPortCertificate(certThumbPrint, configuration.BridgeHttpsPort);
+            CertificateResourceHelpers.EnsureSslPortCertificateInstalled(context.BridgeConfiguration);
```

Comment:
I kept this here because the IResource has no knowledge of the serviceHost, and I feel like shouldn't need to...